src/installation/index.md

```diff
@@ -59,10 +59,12 @@ To build Rust applications for the Espressif chips based on `RISC-V` architectur
 
       This target is currently [Tier 2][rust-lang-book--platform-support-tier2]; note the different flavors of `riscv32` target in Rust covering different [`RISC-V` extensions][wiki-riscv-standard-extensions].
 
-    - For `std` applications, use the target `riscv32imc-esp-espidf`.
-
-      Since this target is currently [Tier 3][rust-lang-book--platform-support-tier3], it does not have pre-built objects distributed through `rustup`, and **it does not need to be installed** as the `no_std` target.
+    - For `std` applications:
 
+      Since this target is currently [Tier 3][rust-lang-book--platform-support-tier3], it does not have pre-built objects distributed through `rustup` and, unlike the `no_std` target, **nothing needs to be installed**. Specify the correct target for your device:
+      
+      - `riscv32imc-esp-espidf` for MCUs which do not support atomics, like ESP32-C2 and ESP32-C3
+      - `riscv32imac-esp-espidf` for MCUs which support atomics, like ESP32-C6, ESP32-H2, and ESP32-P4
```

Comment:
Maybe we could include a reference to https://github.com/Nassiel/rust/blob/1f5361b40c4ccfa2f806cc9f293d8399a5651b9a/src/doc/rustc/src/platform-support/esp-idf.md and avoid listing all the MCUs that every target supports

Reply:
Agree that this book shouldn't be responsible for listing MCUs.
DId you mean to link to the rust book under Nassiel's fork? I assume the master branch of the offical fork is most likely to stay updated.
https://github.com/rust-lang/rust/blob/master/src/doc/rustc/src/platform-support/esp-idf.md
(Any idea why a link like https://doc.rust-lang.org/rustc/platform-support/esp-idf.html doesn't work?)

Reply:
> DId you mean to link to the rust book under Nassiel's fork? I assume the master branch of the offical fork is most likely to stay updated.

Sorry for the typo, I meant the upstream repo.

> Any idea why a link like https://doc.rust-lang.org/rustc/platform-support/esp-idf.html doesn't work?

Not sure how often they regenerate the mdbook, maybe its every release and the file was recently added, so, maybe not yet deployed.

Reply:
Done